### PR TITLE
ssl support

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by tmrlvi** [comment]
Why the choice of `tokio_openssl` over the official `tokio_native_ssl`?

**Comment by splatboydev** [comment]
Also, this does not work.

File: src/client/stratum.rs [hunk 1]
```diff
@@ -169,11 +170,17 @@ impl StratumHandler {
...
         block_template_ctr: Option<Arc<AtomicU16>>,
     ) -> Result<Box<Self>, Error> {
         info!("Connecting to {}", address);
-        let socket = TcpStream::connect(address).await?;
+        
+        if ssl_true == true {
+            let socket = SslStream::connect(address).await?;
+        } else {
+            let socket = TcpStream::connect(address).await?;
+        }
```
Comment (tmrlvi) [inline src/client/stratum.rs:183]: @splatboydev This part doesn't compile, right?

**Comment by tmrlvi** [comment]
I'm rejecting the PR do to incompleteness and low activity.

File: src/client/stratum.rs (hunk 1 above)
Comment (splatboydev) [reply src/client/stratum.rs:183]: Yes, that's correct.

**Pull request closed without merging**